tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartLocaleGrowth.java

```diff
@@ -0,0 +1,347 @@
+package org.unicode.cldr.tool;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+
+import org.unicode.cldr.draft.FileUtilities;
+import org.unicode.cldr.tool.Option.Options;
+import org.unicode.cldr.util.CLDRConfig;
+import org.unicode.cldr.util.CLDRFile;
+import org.unicode.cldr.util.CLDRFile.Status;
+import org.unicode.cldr.util.CLDRPaths;
+import org.unicode.cldr.util.CldrUtility;
+import org.unicode.cldr.util.Counter;
+import org.unicode.cldr.util.Counter2;
+import org.unicode.cldr.util.Level;
+import org.unicode.cldr.util.LocaleNames;
+import org.unicode.cldr.util.PathHeader;
+import org.unicode.cldr.util.PathHeader.Factory;
+import org.unicode.cldr.util.PatternCache;
+import org.unicode.cldr.util.RegexLookup;
+import org.unicode.cldr.util.RegexLookup.LookupType;
+import org.unicode.cldr.util.SimpleFactory;
+import org.unicode.cldr.util.SupplementalDataInfo;
+import org.unicode.cldr.util.VettingViewer;
+import org.unicode.cldr.util.VettingViewer.MissingStatus;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Ordering;
+import com.ibm.icu.impl.Relation;
+import com.ibm.icu.util.VersionInfo;
+
+public class ChartLocaleGrowth {
+
+    private static final boolean DEBUG = true;
+    private static final char DEBUG_FILTER = 0; // use letter to only load locales starting with that letter
+
+    private static CLDRConfig testInfo = ToolConfig.getToolInstance();
+    private static final SupplementalDataInfo SUPPLEMENTAL_DATA_INFO = testInfo.getSupplementalDataInfo();
+
+    private static org.unicode.cldr.util.Factory factory = testInfo.getCommonAndSeedAndMainAndAnnotationsFactory();
+    private static final CLDRFile ENGLISH = factory.make("en", true);
+
+    // added info using pattern in VettingViewer.
+
+    private static final RegexLookup<Boolean> HACK = RegexLookup.<Boolean> of(LookupType.STANDARD, RegexLookup.RegexFinderTransformPath)
```

Comment:
ideally `HACK` should get a new name or a description of how to remedy the hack.

Reply:
needs description of what this is